src/components/v1/V1Project/V1Project.tsx

```diff
@@ -29,11 +30,22 @@ const RelaunchV1ProjectCallout = ({ className }: { className?: string }) => {
   const { isReady, relaunch } = useRelaunchV1ViaV3Create()
   const isLoading = !isReady
 
+  const { currentPayoutMods } = useContext(V1ProjectContext)
+
   const hasUpgradePermission = useV1ConnectedWalletHasPermission(
     V1OperatorPermission.Configure,
   )
 
-  if (isHidden || !hasUpgradePermission) {
+  const projectMigratedToV3 = currentPayoutMods?.some(
+    payout => payout.allocator === V1_V3_ALLOCATOR_ADDRESS,
+  )
+
+  if (
+    projectMigratedToV3 === undefined ||
```

Comment:
**question** why do we need this line?

Reply:
I've added it since at first value comes as undefined and later it's defined as a whole or empty array, therefore banner appears for a second there and it disappears.